nrrd/tests/test_reading.py

```diff
@@ -176,6 +178,9 @@ def test_read_dup_field_error_and_warn(self):
 
         import warnings
         with warnings.catch_warnings(record=True) as w:
+            # Enable all warnings
+            warnings.simplefilter("always")
```

Comment:
Go ahead and remove this line. When running tests on your branch, I noticed I received some warnings that I didn't get before. I finally figured out it's because of this line.

I have an upcoming PR where I enable filters for the entire tests (I put this code at the top of each file). I also will be fixing some of these warnings too.

Reply:
:+1: . This makes the test fail though, but since you're enabling it in another PR I guess it's fine.

Reply:
Gotcha, that's the reason why the one test was failing. I'm curious as to why this started happening only in your PR. I'm sure there's some reason.

Reply:
The reason is mostly likely caused by the fact that we want to trigger the same warning twice (for both 'C' and 'F'). I don't know if it is intended but it seems that Python 2 only triggers warnings once, omitting the warning in subsequent calls.